src/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  newTodoField: React.RefObject<HTMLInputElement>,
+  isAllCompleted: boolean,
+  newTodoTitle: string,
+  addingNewTodo: (event: React.FormEvent) => void,
+  newTitle: (event: string) => void,
+  isAdding: boolean,
+};
+
+export const AppHeader: React.FC<Props> = ({
+  todos,
+  newTodoField,
+  isAllCompleted,
+  newTodoTitle,
+  addingNewTodo,
+  newTitle,
+  isAdding,
+}) => {
+  return (
+    <header className="todoapp__header">
+      {todos.length !== 0 && (
```

Comment:
So you are passing a full array of todos to the component for checking it's length? Could we make it better?